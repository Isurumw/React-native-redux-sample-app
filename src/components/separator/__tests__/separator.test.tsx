import React from 'react';
import {render} from 'react-native-testing-library';

import Separator from '../separator';

describe('render the custom separator', () => {
  it('the separator component renders correctly', () => {
    const component = render(<Separator />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
