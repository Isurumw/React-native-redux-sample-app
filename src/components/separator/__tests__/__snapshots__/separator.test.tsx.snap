// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the custom separator the separator component renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "width": "100%",
      },
      undefined,
    ]
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#D5D8DF",
        "height": 1,
        "opacity": 0.5,
        "width": "100%",
      }
    }
  />
</View>
`;
